app/components/general/components/bc-table/macro.test.js

```diff
@@ -0,0 +1,149 @@
+import { componentTester } from '../../../../testUtils/componentTester';
+
+const setup = {
+  component: {
+    name: 'bcTable',
+    path: 'components/general/components/bc-table/macro.njk',
+  },
+};
+
+const mockContext = {
+  params: {
+    title: 'Manage organisations and users',
+    description: 'Edit existing account information or add new user to an organisation.',
+    columnInfo: [
+      { data: 'Organisation name' },
+      { data: 'ODS code' },
+    ],
+    columnClass: 'nhsuk-grid-column-one-half',
+    data: [
+      [{ data: 'Greater Manchester CCG' }, { data: 'X01' }],
+      [{ data: 'Hampshire CCG' }, { data: 'X02' }]],
+  },
```

Comment:
can we change this test data so it's more generic

Reply:
I don't think it matters at all but if you like

Reply:
yes please even if it is `column 1` or `row 1` kind of data